firmware_mod/www/cgi-bin/action.cgi

```diff
@@ -742,6 +742,26 @@ if [ -n "$F_cmd" ]; then
         fi
         return
         ;;
+     recording_files)
+      for file in $(find /system/sdcard/DCIM/Recording/ -type f)
```

Comment:
We should list the whole DCIM directory. We could write one *_files script per directory or come up with something more generic.
Maybe have a look at one of these suggesting for listing https://stackoverflow.com/questions/18014779/similar-command-to-linux-tree-command-in-adb-shell

Reply:
Yes good idea